acis_thermal_check/state_builder.py

```diff
@@ -0,0 +1,254 @@
+import os
+import Ska.DBI
+from Chandra.Time import DateTime
+from pprint import pformat
+import Chandra.cmd_states as cmd_states
+from acis_thermal_check.utils import globfile, mylog
+
```

Comment:
FYI there is an existing utility similar to your `globfile` in case you need this elsewhere:  https://github.com/sot/Ska.File/blob/master/Ska/File.py#L54
```
fn = Ska.File.get_globfiles(fileglob)[0]  # Returns a list but requires exactly 1 match
```